builtin/logical/pki/path_manage_keys.go

```diff
@@ -23,15 +23,27 @@ func pathGenerateKey(b *backend) *framework.Path {
 				Default:     "rsa",
 				Description: `Type of the secret key to generate`,
 			},
-			"key_bits": {
+			keyBitsParam: {
 				Type:        framework.TypeInt,
 				Default:     2048,
 				Description: `Type of the secret key to generate`,
 			},
+			"managed_key_name": {
+				Type: framework.TypeString,
+				Description: `The name of the managed key to use when the exported
+type is kms. When kms type is the key type, this field or managed_key_id
+is required. Ignored for other types.`,
+			},
+			"managed_key_id": {
+				Type: framework.TypeString,
+				Description: `The name of the managed key to use when the exported
+type is kms. When kms type is the key type, this field or managed_key_name
+is required. Ignored for other types.`,
+			},
 		},
 
 		Operations: map[logical.Operation]framework.OperationHandler{
-			logical.CreateOperation: &framework.PathOperation{
+			logical.UpdateOperation: &framework.PathOperation{
```

Comment:
This was a bit subtle, but Create is only for PUT operations, we want to support POST I believe.